spec.md

```diff
@@ -2109,6 +2239,46 @@ message NodeGetInfoResponse {
 If the plugin is unable to complete the NodeGetInfo call successfully, it MUST return a non-ok gRPC code in the gRPC status.
 The CO MUST implement the specified error recovery behavior when it encounters the gRPC error code.
 
+#### `NodeExpandVolume`
+
+A Node Plugin MUST implement this RPC call if it has `EXPAND_VOLUME` node capability.
+This RPC call allows CO to expand volume on a node.
+
+`NodeExpandVolume` ONLY supports expansion of already node-published or node-staged volumes on the given `volume_path`.
+
+If plugin has `STAGE_UNSTAGE_VOLUME` node capability - `NodeExpandVolume` MUST be called after successful `NodeStageVolume`.
```

Comment:
```
If the plugin has the `STAGE_UNSTAGE_VOLUME` node capability then:
* `NodeExpandVolume` MUST be called after successful `NodeStageVolume`.
* `NodeExpandVolume` MAY be called before or after `NodePublishVolume`.

Otherwise, `NodeExpandVolume` MUST be called after successful `NodePublishVolume`.